client/clientservice/src/event_service.cpp

```diff
@@ -0,0 +1,32 @@
+// Concord
+//
+// Copyright (c) 2021 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License"). You may not use this product except in
+// compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright notices and license terms. Your use of
+// these subcomponents is subject to the terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#include <iostream>
+
+#include "event_service.hpp"
+
+using grpc::Status;
+using grpc::ServerContext;
+using grpc::ServerWriter;
+
+using vmware::concord::client::v1::StreamEventGroupsRequest;
+using vmware::concord::client::v1::EventGroup;
+
+namespace concord::client {
+
+Status EventServiceImpl::StreamEventGroups(ServerContext* context,
```

Comment:
I know that this is just the beginning but would you mind changing the method's name to `camelCase`?
I think that having `camelCase` is the only thing we've managed to reach consensus for :)

Reply:
I cannot. This is the protobuf/gRPC generated interface.

Reply:
Oh, I did not realize that it was part of the interface, my bad!